mantidproject/autoreduce
# Remove SMB dependency

This addresses https://github.com/mantidproject/autoreduce/issues/179. As it was before, the frontend server wrote the script file out to a temp directory on the backend server, and passed the path to this file in the message itself, plus the reduction variables in the message body. Now, the server sends the script in plaintext in the message body. The backend server then parses this string into a Python module, and substitutes variables there. 

For the production servers, this removes the need for an SMB share between the frontend and the backend altogether. In the dev system, the backend still uses the frontend as a dummy for the ISIS archive, but that's irrelevant.

To test
* Run a job that should succeed
* Check that the first few lines of the script are shown in server logs
* Check the run succeeds
